### DISCUSSION TITLE
How to make it work...?

### DISCUSSION THREAD

**Discussion by totte1015** [valinet/ExplorerPatcher#2792] [Q&A] [answered]
Uhhh... So I got recommended Explorer Patcher to get things back to Windows 10, but when opening installer "ep_setup.exe" it just flashes my windows, then back to normal and nothing has changed whatsoever.
What do I do now? Is there no settings or anything?

**Comment by pyrates999** [comment]
Please right click on ep_setup.exe and select properties.  Please show a screen shot of the tabs general and details.

**Comment by totte1015** [comment]
![Screenshot 2024-02-07 125054](https://github.com/valinet/ExplorerPatcher/assets/32680625/b42ec04a-d426-4514-a1f8-09e8e0134b4a)
![Screenshot 2024-02-07 125104](https://github.com/valinet/ExplorerPatcher/assets/32680625/179d5f37-f959-4251-80e6-edaad199e477)

Here ya go, what is it that you are looking for? (Sadly its on swedish but)

**Comment by pyrates999** [comment]
Good, your download is not corrupted.

Is this a work computer?  They might not allow unsigned exe's to run.

**Reply by totte1015** [reply]
Nono, its private 100%, built by me haha

I just tested it today now again after a restart and it does the same thing, start menu and background flashes away for like 3 seconds, then re-appears and nothing more happens. Its a really weird reaction. No message from Windows Defender or anything like that. And I tried running as admin

**Comment by Amrsatrio** [comment, marked as answer]
> Uhhh... So I got recommended Explorer Patcher to get things back to Windows 10, **but when opening installer "ep_setup.exe" it just flashes my windows**, then back to normal and nothing has changed whatsoever.

This is the normal behavior of ep_setup.exe. I think for the ease of development it was made so that there are no feedback after installation. After installing ExplorerPatcher you should see a new entry in the start menu named Properties (ExplorerPatcher) where Properties is in your Windows language during ExplorerPatcher installation.

Or, right click the taskbar and you should see a menu entry named "Properties".

If you are using a personal computer and have admin rights on your current Windows user, EP's installation should perform as normally. Should you have any more problems, feel free to say about it here.

**Reply by totte1015** [reply]
Oh wow, I would have never seen that option, its so hidden lol!

But now I got the settings, nice. I guess the fact it was on my system language fooled me thinking that "Properties" option has always been there.
Thanks!
